configuration/backends/plain/plain-wrapper.sh

```diff
@@ -155,6 +163,26 @@ function log_compiler_call
     esac
   done
 
+  # create the JSON blob to log for each source file for the compilation database
+  local -a COMMAND_ARRAY=("$compiler")
+  COMMAND_ARRAY+=("$@")
+  local JSON_COMMAND
+  JSON_COMMAND="$(printf '%s\n' "${COMMAND_ARRAY[@]}" | jq -R . | jq -sc .)"
```

Comment:
Are we sure `jq` is available? I think we should explicitly check beforehand as the error in the pipeline.

Reply:
I will add a check around the compilation database code, and will not run it, in case "jq" is not around.